TKGame/Entity.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Microsoft.Xna.Framework.Graphics;
+using Microsoft.Xna.Framework;
+
+namespace TKGame
+{
+    abstract class Entity
+    {
+        protected Texture2D image;
+        protected Color color = Color.White;
+
+        public Vector2 Position, Velocity;
```

Comment:
I'm thinking we'll probably want to have a `Transform` struct/class like Unity's `Transform` that holds information like position, scale, rotation: https://docs.unity3d.com/ScriptReference/Transform.html